Compentio.SourceMapper.Tests/DependencyInjections/AutofacProcessorStrategyTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Compentio.SourceMapper.Metadata;
+using Compentio.SourceMapper.Processors.DependencyInjection;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Xunit;
+
+namespace Compentio.SourceMapper.Tests.DependencyInjections
+{
+    public class AutofacProcessorStrategyTests
+    {
+        private const string BUILDER_REGISTER_SERVICE = "builder.RegisterType";
```

Comment:
It is `java` style naming conventions:  https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants

Reply:
I learned C# programming from Code Complete by Steve McConnell (Programming Cookbook from Microsoft )  and I always do this for constants. It is convention, if You would like - I can change this.

Reply:
Please, see the limk from Microsoft